core/src/main/scala/kafka/server/KafkaHealthcheck.scala

```diff
@@ -35,14 +36,15 @@ import org.apache.zookeeper.Watcher.Event.KeeperState
  * Right now our definition of health is fairly naive. If we register in zk we are healthy, otherwise
  * we are dead.
```

Comment:
Thanks for the comment. I would say it's still valid as we don't use the metrics to determine health.